vue-language-tools/vue-language-server/src/languageServerPlugin.ts

```diff
@@ -116,6 +122,13 @@ const plugin: LanguageServerPlugin<VueServerInitializationOptions, vue.LanguageS
 			},
 		},
 	};
+
+	function getVueExts(baseExts: string[]) {
+		return [
+			...baseExts,
+			...initOptions.additionalExtensions?.map(ext => '.' + ext) ?? [],
```

Comment:
This code should not working correctly, currently `initOptions.additionalExtensions` includes dot, so map result is `['..ext1', '..ext2']`

Reply:
I think the fact that this wasn't working was why I originally thought I needed the vueCompilerOptions setting above. However, now that the additionalExtensions VS Code setting no longer has the period this should be correct.